rollbar.gemspec

```diff
@@ -18,4 +18,12 @@ Gem::Specification.new do |gem|
   gem.version       = Rollbar::VERSION
 
   gem.add_runtime_dependency 'multi_json'
+
+  if s.respond_to?(:metadata)
```

Comment:
@viraptor thank you for the PR. `s` isn't defined. Can you change to `gem`?

Reply:
Right, sorry. Silly copy-paste. Fixing now...